### ISSUE TITLE
support github submodules

### ISSUE THREAD

**Issue by mooreatv** [mooreatv/packager#1]
the friendly folks at BigWigs like to reject PRs and delete comments
https://github.com/BigWigsMods/packager/pull/46

so we can probably have a really open version of the packager here or under wowaddons or with @Wutname1

**Comment by mooreatv** [comment]
https://github.com/Wutname1/packager/commit/40e0683ad3f14967af61ec7843a1279d237ee538

**Comment by Wutname1** [comment]
Ya, that was a shame.  I much more prefer submodules for libs that change often and have breaking changes when they change. Then you dont have to rely on the lib having tags for the point that you want to run your project on.

**Comment by mooreatv** [comment]
yup, I think it's reasonable to support both ways (submodules do have some merging/cleanup/maintenance issues too but it's fine if it's setup)
